crates/matrix-sdk/src/authentication.rs

```diff
@@ -60,6 +60,15 @@ impl AuthSession {
             AuthSession::Oidc(session) => &session.user.tokens.access_token,
         }
     }
+
+    /// Get the refresh token of this session.
+    pub fn refresh_token(&self) -> &Option<String> {
```

Comment:
nit: can we rename this `get_refresh_token()`? At first I thought it was a method to refresh a token.

nit: also it's more idiomatic to return `Option<&>` rather than `&Option<...>`, by adding `.as_ref()` calls to the `.refresh_token` statements below. So the method could return `Option<&str>`, because of the implict conversion of `&String` to `&str`.